vcd/resource_vcd_catalog.go

```diff
@@ -123,3 +127,45 @@ func resourceVcdCatalogDelete(d *schema.ResourceData, meta interface{}) error {
 	log.Printf("[TRACE] Catalog delete completed: %#v", adminCatalog.AdminCatalog)
 	return nil
 }
+
+// Imports a Catalog into Terraform state
+// This function task is to get the data from vCD and fill the resource data container
+// Expects the d.ID() to be a path to the resource made of Org name + dot + Catalog name
+//
+// Example import path (id): org_name.catalog_name
+func resourceVcdCatalogImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	resourceURI := strings.Split(d.Id(), ".")
+	if len(resourceURI) != 2 {
+		return nil, fmt.Errorf("resource name must be specified as org.catalog")
+	}
+	orgName, catalogName := resourceURI[0], resourceURI[1]
+
+	vcdClient := meta.(*VCDClient)
+	adminOrg, err := vcdClient.GetAdminOrgByName(orgName)
+	if err != nil {
+		return nil, fmt.Errorf(errorRetrievingOrg, orgName)
+	}
+
+	catalog, err := adminOrg.GetCatalogByName(catalogName, false)
+	if err != nil {
+		return nil, govcd.ErrorEntityNotFound
+	}
+
+	err = d.Set("org", orgName)
```

Comment:
I think we don't need `err` check for d.Set ^^

Reply:
fixed